flowforge/input/Components.py

```diff
@@ -1461,12 +1467,13 @@ def __init__(
             assert np.all(len(map_row) == len(orifice_row) for map_row, orifice_row in zip(self._map, self._orificing))
         for r, col in enumerate(self._map):
             for c, val in enumerate(col):
-                cname = f"{str(val):s}-{r + 1:d}-{c + 1:d}"
-                yc, xc = self._getChannelCoords(r, c)
-                centroids[cname] = [xc, yc]
-                if self._orificing is not None:
-                    self.tmpComponents[str(val)].addKlossInlet(self._orificing[r][c])
-                extended_comps[cname] = deepcopy(self.tmpComponents[str(val)])
+                if val != 0:
```

Comment:
This is necessary because sometimes you might want a hex grid with gaps. We could make it something else, but we do need a way to indicate a gap in a grid.